lib/suspenders/generators/app_generator.rb

```diff
@@ -212,10 +212,6 @@ def outro
       say "Remember to run 'rails generate airbrake' with your API key."
     end
 
-    def run_bundle
-      # Let's not: We'll bundle manually at the right spot
-    end
-
```

Comment:
I think this is meant to over-write `run_bundle` defined in `app_base`

- https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/app_base.rb#L334
- https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/rails/app/app_generator.rb#L259

Although I don't know why you don't just set skip_bundle to true https://github.com/rails/rails/blob/4-2-stable/railties/lib/rails/generators/app_base.rb#L32